packages/foreigntypes/posixtypes.pony

```diff
@@ -0,0 +1,95 @@
+// This file defines the standard Posix type to make FFI calls easier
+// to manage and more consistent.
+
+type BlkCnt is I64
+
+// not consistent across OSX (always 32 bit int)/Linux (Long)
+type BlkSize is ILong
+type OSXBlkSize is I32
```

Comment:
I'm not a big fan of this way of distinguishing between the Linux size and the OSX size.

I'd rather see something like this:
```pony
ifdef linux then
  type BlkSize is ILong
elseifdef osx then
  type BlkSize is I32
end
```

However, that is not currently valid Pony syntax.  I think we should take a look at whether such syntax could be supported, and pursue that alternative.

I suppose this PR could still be merged as-is with this implementation as a stop-gap until we get those ifdefs, but I'll be grumbly about it :disappointed:.